packages/reports/src/Containers/open-positions.tsx

```diff
@@ -488,7 +483,7 @@ const OpenPositions = ({
             if (is_accumulator_selected)
                 return (
                     isAccumulatorContract(contract_info.contract_type) &&
-                    (`${getGrowthRatePercentage(contract_info.growth_rate)}%` === accumulator_rate ||
+                    (`${getGrowthRatePercentage(contract_info.growth_rate || 1)}%` === accumulator_rate ||
```

Comment:
```suggestion
                    (`${getGrowthRatePercentage(contract_info.growth_rate as number)}%` === accumulator_rate ||
```
@henry-deriv as number here is a better option because 1 is invalid growth_rate, and we know that in contract_info will always contain growth_rate for an Accumulator contract anyway :)